### DISCUSSION TITLE
Is salamandra down?

### DISCUSSION THREAD

**Discussion by micro-hg-cpg** [marbits-icm/marbits-public#10] [Q & A]
Hello! I cant connect to salamandra. Is it not working for some reason or is just me?

**Comment by pablo-icm** [comment]
No it is not. But it keeps rejecting connections. I'm asking IT.

**Comment by pablo-icm** [comment]
It seems to be solved. It works for me.

**Comment by franlat** [comment]
Is it down again? I can't seem to connect to it since yesterday.

**Reply by pablo-icm** [reply]
Yes, it looks like you can only access via VPN as of today. The server is up but it rejects ssh access.

**Comment by franlat** [comment]
Solved! Closing it down
